models/point_of_interest.py

```diff
@@ -38,6 +41,31 @@ def ensure_categories_as_list(cls, categories: Any) -> List[str]:
     def ensure_organizations_as_list(cls, organizations: Any) -> List[str]:
         return _convert_to_list(organizations)
 
+    @validator('url', pre=True)
+    def sanitize_url(cls, url: str) -> str:
+        org_url = url
```

Comment:
I suggest extracting this logic into a separate module and importing that function, the constants can be in that module too so that we keep the model more concise, so here it would be just:

```python
@validator('url', pre=True)
def sanitize_url(cls, url: str) -> str:
    return sanitize_url(url)
```

Reply:
Sure.

> extracting this logic into a separate module and importing that function,

Could you demo this structure for me? I didn't get the point.

Reply:
Do you mean 
```python
# services/url_sanitizer.py
def sanitise_url(url: str) -> str:
    ...
```

Reply:
Yes, exactly! Maybe not services (coz those are for IO) but rather `utils/url_sanitizer.py` or something like that

Reply:
Actually, we already have the `validation` package, so maybe `validation/url_sanitizer.py`

Reply:
I had to do a little refactoring in #51 so that it's possible without creating circular imports